manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -80,6 +80,11 @@ rules:
     resources: ["tokenreviews"]
     verbs: ["create"]
 
+  # Used by Istiod to verify gateway SDS
```

Comment:
this is seems out of place - is it a weird github display or something going on here?

Reply:
I need rebase later.